### PR TITLE
add reload selection button on `/`

### PR DESCRIPTION
**Pull request by hugurp**
wish it did not reload the UI where the buttons are, but not sure how.

### CODE REVIEW

File: app/channels/index/controller.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import Controller from '@ember/controller';
+
+export default Controller.extend({
+	actions: {
+		refreshSelection() {
+			console.log('poing')
```
Comment (oskarrough) [inline app/channels/index/controller.js]: remove log

**Comment by oskarrough** [comment]
It's a cool little button! Here it doesn't flicker at all, UI seems good.

On the staging-database it you see the same channels again and again. But on production it feels better. So we should remember to feature enough radios to keep it fresh.

File: app/channels/index/template.hbs [hunk 2]
```diff
@@ -1,33 +1,37 @@
...
+				{{#if session.isAuthenticated}}
+					<p>Start your journey by listening to one of the radio channels here.</p>
+				{{else}}
+					<p>Start your journey by listening to one of the radio channels here, <a href={{href-to "about"}}>learn more</a> or <a href={{href-to "auth.signup"}}>create my radio</a>.</p>
+				{{/if}}
+				<p>
+					{{play-random-channel class="Btn--full"}}
+				</p>
+				<p>
+					<button {{action 'refreshSelection'}} class="Btn Btn--full">⟳ Reload selection</button>
```
Comment (oskarrough) [inline app/channels/index/template.hbs]: wrap icon in `<i>`

**Comment by oskarrough** [comment]
How about "Refresh radios" instead of "Refresh selection"? Also to avoid any confusion with the tracks selection.

**Comment by hugurp** [comment]
What about "Load new radios" ? https://deploy-preview-206--radio4000.netlify.com/

**Comment by oskarrough** [comment]
Cool!

**Comment by oskarrough** [comment]
Merge? @hugurp

**Pull request merged**